src/backend/adapters/base-resource.js

```diff
@@ -33,17 +35,16 @@ class BaseResource {
    * @return {String}         database name
    */
   databaseName() {
-    throw new NotImplementedError()
+    throw new NotImplementedError('BaseResource#databaseName')
   }
 
-  
   /**
    * Returns type of the database. It is used to compute sidebar icon for
-   * given resource
+   * given resource. Default: 'database'
    * @return {String}
    */
   databaseType() {
-    throw new NotImplementedError()
```

Comment:
now databaseType is "database" by default - I changed that because it simplifies adding custom resources (i.e. api wrappers) - developers wont have to implement that